src/Playwright/Transport/Channels/PageChannel.cs

```diff
@@ -105,7 +105,7 @@ internal override void OnMessage(string method, JsonElement? serverParams)
                     Popup?.Invoke(this, new() { Page = serverParams?.GetProperty("page").ToObject<PageChannel>(Connection.DefaultJsonSerializerOptions).Object });
                     break;
                 case "pageError":
-                    PageError?.Invoke(this, serverParams?.GetProperty("error").GetProperty("error").ToObject<SerializedError>(Connection.DefaultJsonSerializerOptions));
```

Comment:
Let's add a test for this.

Reply:
there are tests so I discovered that I had to adjust this to make the test pass.